packages/perseus/src/widgets/grapher.jsx

```diff
@@ -1,6 +1,11 @@
+/* eslint-disable react/prop-types */
 /* eslint-disable react/sort-comp */
```

Comment:
I like this better than a single `eslint-disable` with a bunch of rules because it makes adding/removing rules easier.